doc/source/building/index.rst

```diff
@@ -376,6 +376,37 @@ interface is self-documenting, so please see ``python dev.py --help`` and
 ``python dev.py <subcommand> --help`` for detailed guidance.
 
 
+.. admonition:: IDE support & editable installs
+
+    While the ``dev.py`` interface is our recommended way of working on SciPy,
+    it has one limitation: because of the custom install location, setting
+    breakpoints inside an IDE or debugger does not work as expected. This will
+    only work with an *in-place build* (or "editable install").
+
+    Editable installs are supported. It is important to understand that **you
+    must use either an editable install or dev.py, but not both**. If you use
+    editable installs, you can use ``pytest`` and other development tools
+    directly instead of using ``dev.py``.
```

Comment:
Is there a possibility of updating `dev.py` to also work with development installs?

Reply:
Yes, at some point. But I don't want to do that yet, because there are trade-offs. Things like multiple builds in parallel won't work, working with `__file__` may not work (not entirely sure), and error messages get confusing when you do `git clean -xdf` without `pip uninstal scipy`. So I'd say it's only for folks who really want the "breakpoint in IDE" support; for others `dev.py` as is is nicer.